overwatch/processing/alarms/impl/betweenValuesAlarm.py

```diff
@@ -17,5 +17,5 @@ def checkAlarm(self, trend):
         if self.minVal <= testedValue <= self.maxVal:
             return False, ''
 
-        msg = "value: {} not in [{}, {}]".format(testedValue, self.minVal, self.maxVal)
+        msg = "(BetweenValuesAlarm): value {} not in [{}, {}]".format(testedValue, self.minVal, self.maxVal)
```

Comment:
I was thinking about adding class name to alarm message, it take name from class.__name__, it was general function, but it was difficult to manage, so maybe it is better approach.